pyscriptjs/src/components/pyscript.ts

```diff
@@ -67,6 +67,85 @@ export function make_PyScript(interpreter: InterpreterClient, app: PyScriptApp)
         }
     }
 
+    // bootstrap the <script> tag fallback only if needed (once per definition)
+    if (!customElements.get('py-script')) {
+        // allow any HTMLScriptElement to behave like a PyScript custom-elelement
+        type PyScriptElement = HTMLScriptElement & PyScript;
+
+        // the <script> tags to look for, acting like a <py-script> one
+        const pyScriptCSS = 'script[type="py"],script[type="py-script"]';
+
+        // bootstrap with the same connectedCallback logic any <script>
+        const bootstrap = async (script: PyScriptElement) => {
+            app.incrementPendingTags();
+            let releaseLock: () => void;
+            try {
+                releaseLock = await app.tagExecutionLock();
+                const pyScriptTag = document.createElement('py-script-tag') as PyScript;
+                script.after(pyScriptTag);
+                ensureUniqueId(pyScriptTag);
+                // remove the first empty line to preserve line numbers/counting
+                let src = ltrim(script.textContent.replace(/^[\r\n]+/, ''));
+                const url = script.getAttribute('src');
+                if (url) {
+                    try {
+                        const response = await robustFetch(url);
+                        src = await response.text();
+                    } catch (err) {
+                        const e = err as Error;
+                        _createAlertBanner(e.message);
+                        throw e;
+                    }
+                }
```

Comment:
Seems like we have some duplication here, should we just use the `getPySrc` from line 52?

Reply:
the devil is in the detail ... the current `try/catch` has `this.innerHTML = '';` before throwing ... which is completely meaningless for the proposed `<script>` ... I can surely factor out the `try/catch` and pass along a flag like `cleanHTML` though, let's see how it goes 👍

Reply:
Yeah that makes sense thank you!

Reply:
done